libraries/GCS_MAVLink/MAVLink_routing.cpp

```diff
@@ -130,10 +132,12 @@ bool MAVLink_routing::check_and_forward(GCS_MAVLINK &in_link, const mavlink_mess
         return true;
     }
 
+#if HAL_ADSB_ENABLED
```

Comment:
I guess we could do this:
```suggestion
#if HAL_ADSB_ENABLED && (APM_BUILD_COPTER_OR_HELI || APM_BUILD_TYPE(APM_BUILD_ArduPlane))
```